src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -430,17 +430,15 @@ def update_app_settings(cmd, resource_group_name, name, settings=None, slot=None
         app_settings.properties[setting_name] = value
     client = web_client_factory(cmd.cli_ctx)
 
-
 # TODO: Centauri currently return wrong payload for update appsettings, remove this once backend has the fix.
     if is_centauri_functionapp(cmd, resource_group_name, name):
         update_application_settings_polling(cmd, resource_group_name, name, app_settings, slot, client)
-        result = _generic_site_operation(cmd.cli_ctx, resource_group_name, name, 'list_application_settings', slot)
+        _generic_site_operation(cmd.cli_ctx, resource_group_name, name, 'list_application_settings', slot)
```

Comment:
My previous comment https://github.com/Azure/azure-cli/pull/27557#discussion_r1352498350 was only to suggest not introducing `--show-values`. A lower-impact change is to only redact sensitive data, instead of removing the whole output.

Reply:
Okay, I've made the change to redact app settings (setting to null) for the four without a --show-values parameter